introduction/problems-solved.md

```diff
@@ -0,0 +1,35 @@
+This file holds the text to accompany the [What Problems Does InnerSource Solve?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321607) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What Problems Does InnerSource Solve?
+
+To illustrate the types of situations where inner source can help, imagine the following.
+Suppose two teams at the same company deliver separate pieces of software with one team's software depending on that of the other.
+An example could be a user experience that depends on an API service to retrieve data for display.
+This situation is common in a large enterprise where a single team producing software may have dozens or hundreds of consumers.
+
+When consuming teams need many features, producing teams normally have some sort of requirements and prioritization process to decide which features they will work on.
+For critical feature requests that are not prioritized for immediate work, the consuming team may commonly choose one of 3 options, each of which come with their own drawbacks.
+
+1. **Wait it out**. The consuming team may do nothing and limp along without the requested functionality.
+  This option requires the least amount of work on their side.
+  Depending on the benefit of the feature request, waiting might be just fine.
+  However, it may come with real amounts of pain, especially if the requested functionalty is never delivered.
+1. **Workaround**. A consuming team that doesn't want to wait may do extra work somewhere else to compensate for the absence of their requested feature.
+  This extra work may come as change in the consuming project.
+  Alternatively, they may create a new project that meets their needs and replaces their use of all or part of the producing team's system (code/project duplication).
+  This strategy allows the consuming team to get the requested feature via their own efforts only (no hard dependencies). It comes with several drawbacks, though.
```

Comment:
> (no hard dependencies)

Just wondering: what would be a _soft_ dependency?

Reply:
You're right, that term is not well-defined.  Removed it.